packages/polymath-ui/src/components/Navbar/Navbar.js

```diff
@@ -21,6 +24,8 @@ type StateProps = {|
 |};
 
 const mapStateToProps = (state: RootState): StateProps => ({
+  // TODO: refactor this! We have a cross dependency here, this depends
```

Comment:
This `TODO` fits perfectly with what I mentioned before in this review. Let's just add the author's github handle to it